assets/js/chooser.js

```diff
@@ -138,4 +148,6 @@ $(function() {
         defaultOsChoice = "linux";
     }
     hideShowChoices("os", selectOs, defaultOsChoice);
+    hideShowChoices("cloud", selectCloud, "aws");
+    hideShowChoices("k8s-language", selectK8sLang, "yaml");
```

Comment:
Feels like we should default to the standard Pulumi model, and let folks choose YAML if they want to compare against "classic" -- or Kx if they want to see the nicer variant.

Reply:
Done